lib/Controller/RoomController.php

```diff
@@ -705,6 +705,9 @@ protected function createEmptyRoom(string $roomName, bool $public = true, string
 		} elseif ($objectType === Room::OBJECT_TYPE_PHONE) {
 			// Ignoring any user input on this one
 			$objectId = $objectType;
+		} elseif($objectType === Room::OBJECT_TYPE_EVENT) {
+			// We don't have a UUID for this (yet)
+			$objectId = sha1($this->userId . $this->timeFactory->getTime());
```

Comment:
No, the sending client has to do this

Reply:
you mean in https://github.com/nextcloud/calendar/pull/6768 ?

Reply:
Done.